project-koku/koku
# COST-1014: Keep 3 months of data in materialized views

**Testing**
1. Verify 3 bills of AWS exist
2. Verify AWS materialized views have 3 months of data
3. Verify 3 bills of Azure exist
4. Verify Azure materialized views have 3 months
5. Verify we have 3 OCP report periods
6. Verify OCP has 3 months in materialized views
7. Verify 3 GCP bills 
8. Verify GCP materialized views have 3 months of data

**Test Results**
[cost_1014_ut.txt](https://github.com/project-koku/koku/files/5996804/cost_1014_ut.txt)
